src/templates/duplicate-modal.html

```diff
@@ -17,6 +17,17 @@ <h5 class='modal-title' id='duplicateModalLabel'>{{ 'Duplicate entry'|trans }}</
             <span class='slider'></span>
           </label>
         </div>
+
+      {% if Entity.entityType.value == 'items' or Entity.entityType.value == 'experiments' %}
```

Comment:
always use experiments before items. It's just a convention. It's the natural order: experiments > items > experiments_templates > items_types.